calculator-app/src/Buttonnumbers.js

```diff
@@ -53,14 +53,14 @@ function Buttonnumbers() {
             className="button-cldel"
             onClick={() => setDigit(0)}
           >
-            Clear
+            CLEAR
```

Comment:
use constants variable for hard coded text

Reply:
As per your comments CLEAR and DEL are included in  "constants.js" file.

Reply:
use object like below in constant.js
export const button =  {
  'clear': 'CLEAR',
  'delete': 'DEL'
}

Reply:
As per your comments used object in constants.js